### PR TITLE
[Logs] Tag messages with file name

### PR DESCRIPTION
**Pull request by mnshdw**
### What does this PR do?

This PR adds a `filename` tag to messages sent by the logs agent that contains the name of the file being tailed.

### Motivation

Some customers using wildcards would like to be able to see which file their messages originated from.

### CODE REVIEW

File: pkg/logs/input/tailer/tailer.go [hunk 1]
```diff
@@ -148,6 +152,7 @@ func (t *Tailer) forwardMessages() {
 		origin := message.NewOrigin(t.source)
 		origin.Identifier = identifier
 		origin.Offset = offset
+		origin.SetTags([]string{fmt.Sprintf("filename:%s", filepath.Base(t.path))})
```
Comment (tmichelet) [inline pkg/logs/input/tailer/tailer.go]: A tailer tails a single file, should we generate this tag at tailer startup and avoid doing it for each log line, or would it not matter?

**Review by tmichelet — DISMISSED** [review]

File: pkg/logs/input/tailer/tailer.go (hunk 1 above)
Comment (ajacquemot) [inline pkg/logs/input/tailer/tailer.go]: Wondering whether we should remove the extension too ?

**Review by ajacquemot — DISMISSED** [review]
Let's the rock off 🚀

File: pkg/logs/input/tailer/tailer.go (hunk 1 above)
Comment (tmichelet) [reply pkg/logs/input/tailer/tailer.go]: good point, @NBParis ?

Comment (tmichelet) [reply pkg/logs/input/tailer/tailer.go]: great thanks!

**Review by tmichelet — APPROVED** [review]

**Pull request merged**